### PR TITLE
Include signatory name in decision notice

### PR DESCRIPTION
**Pull request by allbecauseyoutoldmeso**
### Description of change

Include signatory name in postal address rendered in decision notice.

### Story Link

https://trello.com/c/H0B3Uvwa/946-directors-name-missing-from-decision-notice

### Screenshot

Screenshot 2022-05-23 at 14 35 26

### CODE REVIEW

File: app/models/local_authority.rb [hunk 1]
```diff
@@ -5,4 +5,8 @@ class LocalAuthority < ApplicationRecord
   has_many :planning_applications, dependent: :destroy
 
   validates :council_code, presence: true
+
+  def signatory
+    "#{signatory_name}, #{signatory_job_title}"
+  end
```
Comment (allbecauseyoutoldmeso) [inline app/models/local_authority.rb:11]: Not 100% sure about 
1. Whether a view helper would be more appropriate than a method in the model.
2. Whether this should be called `#signatory_name_and_job_title` instead!

Comment (benbaumann95) [reply app/models/local_authority.rb:11]: We could probably update the email text to use this method now. I think they all currently sign off with `@planning_application.local_authority.signatory_name %>, <%= @planning_application.local_authority.signatory_job_title`

Comment (allbecauseyoutoldmeso) [reply app/models/local_authority.rb:11]: I've actually just updated them all to use `<%= @planning_application.local_authority.name %>` instead 😞  😆

File: spec/system/planning_applications/determine_application_spec.rb [hunk 2]
```diff
@@ -67,6 +67,14 @@
           href: decision_notice_planning_application_path(planning_application)
         )
 
+        click_link("View decision notice")
+
+        expect(page).to have_content(
+          "Certificate of Lawful Use or Development Granted"
+        )
+
+        expect(page).to have_content(default_local_authority.signatory)
```
[the remark this answers is not in the document]
Comment (allbecauseyoutoldmeso) [reply spec/system/planning_applications/determine_application_spec.rb]: Good shout!

**Review by benbaumann95 — APPROVED** [review]

**Pull request merged**